configs/defs/fallback.textproto

```diff
@@ -9,6 +9,7 @@ benchmark_configs: {
   id: "cramfs-mounting-disabled-fallback"
   compliance_note: {
     version: { cpe_uri: "fallback" version: "2.0.0" benchmark_document: "CIS Distribution Independent Linux" }
+    version: { cpe_uri: "cpe:/o:centos:centos:7.*" version: "3.1.2" benchmark_document: "CIS Centos Linux 7.x" }
```

Comment:
We'd like to keep the fallback benchmarks defs separate from the rest and only use it in the fallback proto (even if that causes some duplication).

Can you use equivalents from the generic_linux benchmarks here instead? If they don't exist, feel free to copy them over from fallback.textproto into generic_linux.textproto and just remove the "-fallback" suffix from the ID.

Reply:
There are around 60 checks that needs to be copied from fallback to generic, would it be ok anyway?

Reply:
Yeah, that's fine. Just one thing to note: The fallback checks were made to work on things like binary-only containers that don't have a filesystem. Thus the checks pass if the checked file doesn't exist (e.g. see the `aide-installed-fallback` check). If any of these are copied into the generic linux checks they should not have these check alternatives as the generic linux checks are meant to be run on full-fledged linux systems.